### DISCUSSION TITLE
Redirecting after login to previously visited page

### DISCUSSION THREAD

**Discussion by jamiegood** [gladly-team/next-firebase-auth#412] [Q&A] [answered]
Hi there, 

thanks for the great package! Really appreciated. 

I'd like to ask how I can achieve the following with possibly the appPageURL and authPageURL
I have a nextJS app

My scenario: 
So someone arrives at a url on my website. They are not logged in and this page uses withAuthUserSSR 
They arrive at this url: 
mywebsite.com/account/profile/invite

So they get redirected to my Login page
mywebsite.com/login

However once they login I'd like they to get redirected back to THE PREVIOUS PAGE
Redirected after login to:
mywebsite.com/account/profile/invite

That's my scenario I am trying to figure out. 

Would this be possible do you think? 

Many thanks 
Jamie

**Comment by kmjennison** [comment, marked as answer]
Yes, this is possible. Check out documentation here: https://github.com/gladly-team/next-firebase-auth#dynamic-redirects

The [demo app](https://github.com/gladly-team/next-firebase-auth#demo) uses this behavior.

**Reply by jamiegood** [reply]
Thanks very much. Worked a treat.
